### ISSUE TITLE
Should we rename the package or any functions?

### ISSUE THREAD

**Issue by sfirke** [sfirke/tabyl#1]
I kind of think not because:

1. I don't have any better ideas than `tabyl` or `adorn_`
2. It would potentially cause work for people to update their code and brainpower to consider the "new" functions
3. I don't have breaking changes in mind for the functions

But this is the time, if ever, so wanted to put it out there.

**Comment by billdenney** [comment]
As a not big tabyl user (though currently the biggest code contributor to this repo because I'm cheating by copying everything over from `janitor`), I don't have a strong opinion. In the initial PR, I:

* Copied all the `tabyl` and `adorn` functions over along with their support functions
* Copied over all of the tests and the vignette
* Made one code change so that it uses `janitor::round_half_up` with the redirection to `janitor`
* Removed all the deprecated functions since they'll no longer apply

I lean toward keeping it the same so that code still works. Otherwise, users would need to make code changes. If changes were made to function names, I would keep the old ones for backward compatibility. And, only the old ones could be re-imported into `janitor`.

**Comment by gvelasq** [comment]
I'm very glad to see this is in the works!

My development of https://github.com/gvelasq/tidytab continues but as limited by time — it's unlikely to ever be published on CRAN. I'm biased, but `tidytab::tab()` is short and sweet and surprisingly doesn't conflict too much with functions like `base::table()` or `base::tabulate()`. I believe some of the original intent behind `janitor::tabyl()` was to avoid name conflicts?

My two cents is that RStudio doesn't appear to begin autosuggesting functions until one has already typed three characters. So, a 2-4 character function name is likely ideal. It will suggest loaded packages before base. So, `tabl()` or `tab()` or `tb()` could work, for example.

My only issue with `tabyl()` is that it requires a bit of dexterity on QWERTY (right index finger raised to the `y`) and as a console-output function, it's one you want to be able to type in quickly.

**Comment by billdenney** [comment]
I think that this could be pushed to CRAN now, unless there is a desire to make a big change.

There is only one line of code not tested, and it's in the print method (less important, IMO).

Before pushing to CRAN, the version should be updated (to 1.0.0, IMO).

**Comment by billdenney** [comment]
@gvelasq, I realized that my comment last night may have seemed dismissive. That wasn't my intent.

The background is: The `janitor` package needs to be sent to CRAN before the end of the year, and that's going to be very tight with CRAN review timelines. Splitting of `tabyl` from `janitor` has been a long-term goal, and I thought that it would be good to get into this release, if possible.

Renaming functions would be possible with this split, and if there is a reason to do so, as @sfirke says, this is the time to do so.

**Comment by gvelasq** [comment]
Sounds good and no problem at all!

**Comment by billdenney** [comment]
FYI, @sfirke took the pressure off by making a `janitor` release with the minimum required to stay on CRAN.

I just did a search to see if there would be any conflicts from the names `tabl()` or `tab()` from other packages. I found that `tabl` would conflict with the [`gistr`](https://www.rdocumentation.org/packages/gistr/versions/0.9.0/topics/tabl) package, and there is a package called [`tab`](https://cran.r-project.org/package=tab) and a function [`statar::tab()`](https://www.rdocumentation.org/packages/statar/versions/0.7.6/topics/tab).

My preference unless there is a clear, non-conflicting name to change functions to would be to keep functions named as-is.

And as a workaround if we don't make a change any user could run a command like the following to make `tab` work as `tabyl`:

```
library(tabyl)
tab <- tabyl
```
